lib/msf/core/modules/external/go/src/metasploit/module.go

```diff
@@ -0,0 +1,166 @@
+package metasploit
+
+import (
+	"bufio"
+	"encoding/json"
+	"log"
+	"os"
+)
+
+type response struct {
+	Jsonrpc string `json:"jsonrpc"`
+	Id      string `json:"id"`
+}
+
+func rpc_send(res interface{}) {
+	res_str, _ := json.Marshal(res)
+	f := bufio.NewWriter(os.Stdout)
+	defer f.Flush()
```

Comment:
What would we do with the error? If we cannot send, we can only die, since we cannot bubble the error up (it would mean MSF has died). That seems like the best approach we can do.

Reply:
oh ok missed that